libs/wire-api/src/Wire/API/User/Auth.hs

```diff
@@ -373,23 +402,16 @@ instance FromJSON RemoveCookies where
 -- | A temporary API access token.
 data AccessToken = AccessToken
   { user :: UserId,
-    access :: LByteString, -- accessTokenValue
-    tokenType :: TokenType, -- accessTokenType
-    expiresIn :: Integer -- accessTokenExpiresIn
+    -- | assumed to be valid UTF-8
```

Comment:
```suggestion
    -- | TODO: must be valid UTF-8, encode that in the type!
```

Reply:
I applied most of these suggestions, but I might change the TODO to a FUTUREWORK.